src/pybind/mgr/prometheus/module.py

```diff
@@ -1267,13 +1266,19 @@ def _get_pool_info(pool: Dict[str, Any]) -> Tuple[str, str]:
             if 'options' in pool:
                 compression_mode = pool['options'].get('compression_mode', 'none')
 
+            storage_type = {'rbd': 'Block', 'cephfs': 'Filesystem', 'rgw': 'Object'}
+            application_metadata = pool.get('application_metadata', {})
+            application_metadata_str = ', '.join(storage_type.get(str(k)) or str(k)
+                                                 for k in application_metadata)
```

Comment:
if enum then something like this maybe?

```suggestion
            application_metadata = pool.get('application_metadata', {})
            application_metadata_str = ', '.join(
                   StorageType[k].value if k in StorageType.__members__ else str(k)
                   for k in application_metadata
             )
```